README.md

```diff
@@ -17,13 +17,14 @@ The Golden Config plugin is a Nautobot plugin that provides a NetDevOps approach
 
 ### Key Use Cases
 
-This plugin enable five (5) key use cases.
+This plugin enable six (6) key use cases.
 
 1. **Configuration Backups** - Is a Nornir process to connect to devices, optionally parse out lines/secrets, backup the configuration, and save to a Git repository.
 2. **Intended Configuration** - Is a Nornir process to generate configuration based on a Git repo of Jinja files to combine with a GraphQL generated data and a Git repo to store the intended configuration.
 3. **Source of Truth Aggregation** - Is a GraphQL query per device that creates a data structure used in the generation of configuration.
 4. **Configuration Compliance** - Is a process to run comparison of the actual (via backups) and intended (via Jinja file creation) CLI configurations upon saving the actual and intended configuration. This is started by either a Nornir process for cli-like configurations or calling the API for json-like configurations
-5. **Configuration Postprocessing** - (beta) This process renders a valid configuration artifact from an intended configuration, that can be pushed to devices. The current implementation renders this configuration; however, **it doesn't push it** to the target device.
```

Comment:
It still does post processing so can we keep this? Maybe @joewesch can weigh in on proper verbiage

Reply:
Yes, configuration post-processing still exists. It's not enabled for config plans just yet either, so the verbiage is still mostly true that you can't push a config that has been processed.